.travis.yml

```diff
@@ -1,7 +1,12 @@
 language: node_js
+node_js: node # needs to be explicitly specified for nvs on Windows
```

Comment:
Will this still use the version specified in `.nvmrc`?

Reply:
Oh. Probably no.

Reply:
Hmm, I wonder what's the best way forward then... `nvs` (which Travis uses as a replacement for `nvm` on Windows) doesn't read `.nvmrc`... Should I duplicate the version in the Travis config then?

Reply:
Looks like `nvs` looks for file `.node-version` as per https://github.com/jasongin/nvs/issues/113, and I could try to just symlink one into another and add that to the repo... But feels hacky.

Reply:
Hacky indeed but I’d prefer that over duplicating the version number in two places.

Reply:
Would we also have to update the README to add a Windows alternative to `nvm use`?

Reply:
Ugh they're not joking when they say Windows is in beta. Looks like that won't work either, because they use `nvs add` and `.node-version` is read only with `nvs auto`, and customising this command requires basically removing `language:` and writing completely custom install steps in `.travis.yml`. I'm not sure that ends up easier than duplicating version number.

Reply:
> Ugh they're not joking when they say Windows is in beta.

In that case, let's not bother with Windows support on CI for now.

Reply:
Done, I left it commented out with an explainer to make tracking of upstream issue easier. Also left macOS CI enabled just in case.